### PR TITLE
ADEN-5836 Removed old Rubicon integration

### PR DESCRIPTION
**Pull request by nandy-andy**
Cleaning up after integrating Rubicon via prebid.js

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/a1ee61f955b18afd980ac1a74214d58737dcb46d/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/a1ee61f955b18afd980ac1a74214d58737dcb46d/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1123 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js [hunk 1]
```diff
@@ -61,9 +61,9 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 			'bidder_won_price': bidderWon ? realSlotPrices[bidderWon] : '',
 			'bidder_1': transformBidderPrice('indexExchange', realSlotPrices, slotPricesIgnoringTimeout),
 			'bidder_2': transformBidderPrice('appnexus', realSlotPrices, slotPricesIgnoringTimeout),
-			'bidder_3': transformBidderPrice('fastlane', realSlotPrices, slotPricesIgnoringTimeout),
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js:64]: I believe we're using it in prebid and it's working properly

File: extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js [hunk 2]
```diff
@@ -61,9 +61,9 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
...
+			'bidder_3': 'DEPRECATED',
 			'bidder_4': transformBidderPrice('rubicon', realSlotPrices, slotPricesIgnoringTimeout),
-			'bidder_5': transformBidderPrice('fastlane_private', realSlotPrices, slotPricesIgnoringTimeout),
+			'bidder_5': 'DEPRECATED',
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js]: Can we remove `bidder_5` line? `rubicon_fastlane_private` column is going to be renamed and used for other data

File: extensions/wikia/AdEngine/js/ml/hivi/leaderboardInputParser.js [hunk 3]
```diff
@@ -46,8 +46,8 @@ define('ext.wikia.adEngine.ml.hivi.leaderboardInputParser', [
 			articleHeight,												// Article height
 			getBidderPrice('indexExchange', prices),					// IndexExchange
 			getBidderPrice('appnexus', prices),							// AppNexus
-			getBidderPrice('fastlane', prices),							// Rubicon Fastlane
-			getBidderPrice('fastlane_private', prices),					// Rubicon Fastlane private
+			'DEPRECATED',												// Rubicon Fastlane
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/ml/hivi/leaderboardInputParser.js]: It should not contain strings. Please set `0`

File: extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js (hunk 1 above)
Comment (nandy-andy) [reply extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js:64]: We've just checked it. It's going out! :japanese_ogre: We've got `rubicon` and `rubicon_display` for Rubicon via prebid.js.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/2467af302091031c304c8a2a2f765a3ac71f08a5/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/2467af302091031c304c8a2a2f765a3ac71f08a5/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1123 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by fraszczakszymon — APPROVED** [review]

File: extensions/wikia/AdEngine/js/ml/hivi/leaderboardInputParser.js [hunk 4]
```diff
@@ -46,8 +46,8 @@ define('ext.wikia.adEngine.ml.hivi.leaderboardInputParser', [
 			articleHeight,												// Article height
 			getBidderPrice('indexExchange', prices),					// IndexExchange
 			getBidderPrice('appnexus', prices),							// AppNexus
-			getBidderPrice('fastlane', prices),							// Rubicon Fastlane
-			getBidderPrice('fastlane_private', prices),					// Rubicon Fastlane private
+			0,															// Rubicon Fastlane
+			0,															// Rubicon Fastlane private
```
Comment (v3rth) [inline extensions/wikia/AdEngine/js/ml/hivi/leaderboardInputParser.js:50]: Let's add here comment, why 0-s are here and what was here before

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/6de0bc1d0ef53e50140490c334ff1ea54b8db9b2/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/6de0bc1d0ef53e50140490c334ff1ea54b8db9b2/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1123 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/7c0e9a698cd27b183c1b9981655893d7706a7a02/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/7c0e9a698cd27b183c1b9981655893d7706a7a02/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1123 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/cf670ec644c9c7f3a38e4ae5535c68f13e643653/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/cf670ec644c9c7f3a38e4ae5535c68f13e643653/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1123 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/30cae608338704351e79dd56ef407d10cf94502a/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/30cae608338704351e79dd56ef407d10cf94502a/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1123 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**